web-ui/src/main/resources/catalog/templates/admin/harvest/type/filesystem.js

```diff
@@ -53,7 +54,8 @@ var gnHarvesterfilesystem = {
                 + '    <checkFileLastModifiedForUpdate>' + h.site.checkFileLastModifiedForUpdate + '</checkFileLastModifiedForUpdate>'
                 + '    <directory>' + h.site.directory + '</directory>'
                 + '    <icon>' + h.site.icon + '</icon>'
-                + '  </site>' 
+                + '    <beforeScript>' + h.site.beforeScript + '</beforeScript>'
```

Comment:
asking @tonio: the buildResponse is meant to be posted to a controller server-side, so it should not hurt yet (it won't finish in the DOM before getting through a webservice).